### PR TITLE
See list of forms

### PR DESCRIPTION
**Pull request by yohanology**
User can see the list of his forms after a successful login

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -35,6 +41,21 @@ angular.module('starter.controllers', [])
     validateUser();
   };
 
+})
+
+.controller('FormsCtrl', function($scope, $auth, $window, $http) {
+  
+  var user_id = $scope.currentUser.id;
+
+  var url = 'http://localhost:3000' || 'https://formidableforms.herokuapp.com';
```
Comment (h8rry) [inline www/js/controllers.js]: You can use `app.constant` in `app.js` so that you don't have to repeat this again and again

**Comment by h8rry** [comment]
@yohanology Please review

**Comment by yohanology** [comment]
done!

**Comment by yohanology** [comment]
Can merge?

**Pull request merged**